README.rst

```diff
@@ -161,9 +161,12 @@ to configuration.
 Documentation
 -------------
 
-Full documentation including API reference can be found at
+Full documentation can be found at
 `kqueen.readthedocs.io <http://kqueen.readthedocs.io>`__.
 
+API reference is defined at `api.yml <https://github.com/Mirantis/kqueen/blob/master/kqueen/blueprints/api/api3_0.yml>`_
+and swagger UI is available at ``<kqueen_api_url>/api/docs``
```

Comment:
"swagger"? What do you mean? If this is a proper noun, then it should be capitalized. If not, then let's find a better word